docs/Project-Configuration/Project-Commands.md

```diff
@@ -2,6 +2,26 @@
 
 Project Commands are the fundamental units of functionality in an Evergreen task.
 
+# Basic Command Structure
+
+```yaml
+- command: shell.exec
+  display_name: run my cool script ## optional 
+  type: system ## optional
+  timeout_secs: 10 ## optional
+  params:
+    script: echo "my script"
```

Comment:
Should we also document [`retry_on_failure`](https://docs.devprod.prod.corp.mongodb.com/evergreen/Project-Configuration/Project-Configuration-Files#auto-restarting-tasks-upon-failure)?